src/styles/themes/_default-theme.scss

```diff
@@ -165,7 +165,7 @@
     --grey-color-100: #05070a;
     --grey-color: #343c4c;
     --background-color: #fff;
-    --border-color: #bec2ca;
+    --border-color: #69717f;
```

Comment:
Is it really the case that we want to change all usage of var(--border-color)? There are 25 instances of this variable in octuple codebase

this has far reaching impact that goes beyond the scope of pagination buttons. This probably needs to be done with more precision. See my comment on https://github.com/EightfoldAI/vscode/pull/33155

Reply:
@ktorlakonda-eightfold Yair was referring specifically to the Input component border color and isn't relevant to any other component. This line needs to be reverted.

Reply:
Please move this change to a local variable in `src/components/Inputs/input.module.scs`s and ensure it only effects the default border color for `rectangle`, `pill` and `underline` variants and you should be good.

Reply:
updated to use local variable
cc: @dkilgore-eightfold